### DISCUSSION TITLE
Blender 4.0.2 GLTF Export not displaying Transmission

### DISCUSSION THREAD

**Discussion by photoMike** [google/model-viewer#4620] [Q&A] [answered]
Please bear with me as I'm not a developer.

I've attached a glb export from Blender 4.0.2 that has an image as plane, and a cube with a principaled bsdf material. The only changes to the Principaled BSDF I made was to set Transmission to 1 and roughness to 0. 
![image](https://github.com/google/model-viewer/assets/88511475/dfc6aedb-e006-441c-b1b3-378c761c9144)
I then exported to glb using the standard settings. I've attached it as a zip file: Tappy.zip
[Tappy.zip](https://github.com/google/model-viewer/files/13854692/Tappy.zip)

When I view the glb in model viewer the "glass" cube isn't transmitting light. When I use the Khronos.org viewer it does. 

https://github.com/google/model-viewer/assets/88511475/e0d9e71f-ce81-4754-b6f2-2bd2cc8e0fda

Can someone help me understand what I am doing wrong? How do I get the transmission to work? 

Regards,
Mike

**Comment by arpu** [comment]
same proplem  https://gltf-viewer.donmccurdy.com/  ( threejs r160 ) not showing the image

**Comment by WadeMorrison** [comment, marked as answer]
Hey there @photoMike!

As far as I know, different engines render transmission differently. Playing around with your attatched model, I found that your "Tappy285" material's "Blend Mode" was set to "Alpha Blend" instead of "Opaque". Not 100 percent on why this works in some viewers and not in others. But switching that seemed to have solved the issue you were having (at least for Model-Viewer/Three.js)

[Tappy_Updated.zip](https://github.com/google/model-viewer/files/13865305/Tappy_Updated.zip)

Hope that helps!
